CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
-	|
-	|
-	V
-	The Trail becomes bright against a monochrome background of edged shades of 
-	grey. You can see marks and paths that others simply cannot. +50% to a tracking 
-	check, or maximum 2 turns to pass. Passively, gain the ability to sense where your
-	Marked Quarry is, within 20 meters of you.
-	----------------------
-	Highlight Quarry - 
-	If your Quarry is marked, for 1 action, you can activate a Marked Quarry's nanites to make them 
-	more visible. All damage dealt to this target is increased by 10. Two uses per encounter, one use
-	per Quarry. Maximum of Lvl number uses per day. Lasts for the duration of the encounter.
-	At level 10, all damage dealt to this target is increased by 20 (instead of 10).
-	----------------------
-	They Never Saw it Coming -
-	If your Quarry has not seen you for at least 1 turn, your next attack against them does +30 bonus damage.
-	Triggerable once per Quarry.
-	Increases to +50 bonus damage at level 10.
-======================================
+
+Tracker:
+	As an Action, you can study the position and disposition of a target that
+you can see, and indicate it to your Party Members that can communicate with 
+you, either by being on comms with you (see the headset and similar items in
+the "Items/07_Items.txt" document) or that are within 5m of you. Party Members
+(including yourself) that shoot at an indicated target get a +1 to accuracy 
+rolls to hit that target. A target remains indicated until they move, or for
+two Rounds, whichever comes first. You also get 6 skill points in Investigation
+when you take this Class Feat. 
+
+Spotter:
+	Requires that your character be level 10 and have the "Tracker" class feat. 
+The first time each turn that you track an enemy that you have already tracked 
+this encounter, you may do so as a Free Action instead of an Action.
+	For example, let's say a Hunter is fighting an enemy robot and an enemy 
+soldier. The first turn, the Hunter spends one Action to track the robot. The
+second turn, the Hunter spends an Action to track the soldier. The third turn 
+the Hunter tracks the robot again, but this time it is a Free Action, so the 
+Hunter still has 4 Actions left on his turn. Continuing his third turn, the 
+Hunter also tracks soldier. Even though the Hunter already tracked the soldier 
+before, the hunter has used up his one Free Tracking Action this turn, and 
+tracking the soldier again costs one Action.
+	In addition, teammates now get a bonus to the range of rolls which 
+critically hit on tracked enemies that they hit in accordance to their range and 
+weapon type. This is accomplished by you calling out weak spots that you see on 
+the target and describing shooting conditions such as the bullet drop due to 
+humidity, wind speed and direction, and the coriollis effect making the bullet 
+swerve. The bonus to critically hit is listed below. Note that if the chance to 
+critically hit exceeds the chance to hit, it does not increase the chance to 
```

Comment:
The language about critical bonuses is confusing. 
I think an example may help. The math does not seem to be laid out, but other parts of critical range bonuses are here. I think there should be a reference to the section of the rules that involve critical range increases, so this can be condensed.

Reply:
There does not currently exist good language in our rules for me to hook into. This is as good as it's gonna get.